python/hsfs/storage_connector.py

```diff
@@ -180,6 +180,22 @@ def connector_options(self) -> Dict[str, Any]:
         """
         return {}
 
+    def get_generated_feature_groups(self):
```

Comment:
similar to previous comment.
can we rename this to `get_feature_groups_provenance` or `get_generated_feature_groups_provenance` - I would argue in this case `generated` same as `child` is not really needed, you need this when you can have child and generated of same type (such as with fgs) or when the name clashes with something else.
And also add a wrapper named `get_feature_groups` that return accessible and if something in deleted or innaccessible, maybe log this, so it is available in job logs/notebook (info)